src/procedures/proc_wcc.c

```
@@ -0,0 +1,345 @@
+	_process_yield(pdata, yield);
+
+	// save private data
+	ctx->privateData = pdata;
+
+	//--------------------------------------------------------------------------
+	// run the WCC algorithm
+	//--------------------------------------------------------------------------
+
+	GrB_Info info = WCC(&pdata->components, &pdata->N, g, lbls, array_len(lbls),
+			rels, array_len(rels));
+
```

Cppcheck (2.10-2)

[error] Common realloc mistake

(memleakOnRealloc)